packages/core/src/codewhispererChat/commands/registerCommands.ts

```diff
@@ -37,73 +36,13 @@ export const focusAmazonQPanelKeybinding = Commands.declare('_aws.amazonq.focusC
     await focusAmazonQPanel.execute(placeholder, 'keybinding')
 })
 
-const getCommandTriggerType = (data: any): EditorContextCommandTriggerType => {
```

Comment:
I am assuming this fixes the double registering that was happening before?

Reply:
yeah I think what was happening was we were creating the commands here + the commands from commands.ts